R/derive_adeg_params.R

```diff
@@ -151,19 +151,31 @@ derive_param_qtc <- function(dataset,
     get_unit_expr = !!get_unit_expr
   )
 
-  derive_param_computed(
-    dataset,
-    filter = !!filter,
-    parameters = c(qt_code, rr_code),
-    by_vars = by_vars,
-    set_values_to = exprs(
-      AVAL = compute_qtc(
-        qt = !!sym(paste0("AVAL.", qt_code)),
-        rr = !!sym(paste0("AVAL.", rr_code)),
-        method = !!method
-      ),
-      !!!set_values_to
-    )
+  withCallingHandlers(
+    derive_param_computed(
+      dataset,
+      filter = !!filter,
+      parameters = c(qt_code, rr_code),
+      by_vars = by_vars,
+      set_values_to = exprs(
+        AVAL = compute_qtc(
+          qt = !!sym(paste0("AVAL.", qt_code)),
+          rr = !!sym(paste0("AVAL.", rr_code)),
+          method = !!method
+        ),
+        !!!set_values_to
+      )
+    ),
+    derive_param_computed_all_na = function(cnd) {
```

Comment:
Will this message appear every time for the user even if they expect it?

Reply:
Can we add option so user can suppress message if it is expected?

Reply:
Technically, this would be no issue.

But do we really want to add an argument for this?

I would expect that the message is expected only temporary, e.g., at the beginning when only few data is available. For the final delivery it is unlikely to be expected. Why would you define and implement a parameter where you don't expect any records?

I have updated the code such that the class of the message is set. Then you can catch and muffle it easily with `withCallingHandlers()`.

What do you think?

Reply:
> I just get overwhelmed in my programs as I get a gazallion messages from all the different admiral, dplyr, metatools, xportr functions

Where do you get all these messages from?
We use these packages in the admiralroche templates but they produce only very few messages.

Reply:
I am being silly - 4 messages seem like too many!!  Sorry my hyperbole/dramatism doesn't translate well in a PR